docs/FORGER.md

```diff
@@ -65,10 +65,15 @@ This directory will be mounted into the zend container and used to seed the node
 
```

Comment:
`docker compose exec zend gosu user zen-cli getblockcount` you need to be in the right directory in order to execute " docker compose exec" command and now you've removed the "cd" statement. Let's add the "-f" parameter also here, like `docker compose -f deployments/forger/[eon|gobi]/docker-compose.yml exec zend gosu user zen-cli getblockcount`